src/index.ts

```diff
@@ -413,9 +415,6 @@ export class Bigtable {
       }
     }
 
-    const defaultBaseUrl = 'bigtable.googleapis.com';
-    const defaultAdminBaseUrl = 'bigtableadmin.googleapis.com';
```

Comment:
These variables aren't used anymore. These constants also exist in the Gapic layer where the service path is built downstream.